lib/active_record_shards/connection_specification.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-class ActiveRecord::Base
+ActiveRecord::Base.class_eval do
```

Comment:
> no accidental override

Please clarify.

> silences rubocop warning about class definition style (compact vs nested)

This would also be accomplished with:
```rb
module ActiveRecord
  class Base
    # evil stuff
  end
end
```